lib/screens/v2/screen_data/parameters.ex

```diff
@@ -38,7 +41,8 @@ defmodule Screens.V2.ScreenData.Parameters do
     gl_eink_v2: 0,
     solari_v2: 0,
     solari_large_v2: 0,
-    pre_fare_v2: 0
+    pre_fare_v2: 0,
+    dup_v2: 0
```

Comment:
This list is used to set a default value for configs that don't define `interval_offset_seconds`. The value itself is used to offset audio by a defined number of seconds. When we first implemented audio, we got feedback saying that it was sometimes hard to hear the audio because the screen on the other side of the street was drowning it out. So with this, one screen will play audio and the other will play audio `now + interval_offset_seconds` later.

Reply:
I wonder if we could get rid of this map and just return 0 in the [fallback clause](https://github.com/mbta/screens/blob/0c50bc4a689ac6eb78991a4adab38208c1131747/lib/screens/v2/screen_data/parameters.ex#L84-L86) of `get_audio_interval_offset_seconds`? Currently it looks up the value in this map, but that value is always 0.

Reply:
That's a good idea. Makes the function returns more clear.